bbc/simorgh
# Update Burmese and Urdu nav to add topics

Resolves JIRA n/a

Overall changes
======
Update the Burmese nav to replace a FIX and the Sport topic. Update Urdu nav to add the Election 2023 topic  

Code changes
======
- edited navigation section of src/app/lib/config/services/burmese.ts
- edited navigation section of src/app/lib/config/services/urdu.ts
- updated spapshots


Testing
======
1. run locally and open /burmese 
2. the fourth link on the nav (ဆောင်းပါး) should point to /burmese/topics/cl3rq8rkqgxt instead of /burmese/in-depth-54539126
3. the fifth nav link (အင်တာဗျူး) should point to /burmese/topics/cdg42x4kek0t
4. open /urdu the third link in the nav (الیکشن 2023) should point to /urdu/topics/cynd7qxprq0t


Helpful Links
======


[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)
